tests/integration/service/test_feature_table_cache.py

```diff
@@ -237,6 +239,118 @@ async def test_update_feature_table_cache(
     )
 
 
+@pytest.mark.asyncio
+async def test_target_table_cache(
+    feature_store,
+    session,
+    data_source,
+    event_view,
+    observation_table,
+    feature_store_service,
+    observation_table_service,
+    feature_table_cache_service,
+    feature_table_cache_metadata_service,
+    source_type,
+    persistent,
+):
+    """Test target table cache"""
+    feature_store_model = await feature_store_service.get_document(document_id=feature_store.id)
+    observation_table_model = await observation_table_service.get_document(
+        document_id=observation_table.id
+    )
+
+    # add target column in a hacky way - there is no good way to add target from the integration test
```

Comment:
Why is that? Is it possible to write the test using only SDK to demonstrate an actual workflow where users will encounter this issue?

Reply:
No it is not possible. I didn't find the way to add target values to observation table other than via UI/API -> worker job. Workers are not available in integration tests as I know, please correct me if I'm wrong